frontend/summary/summaryTable/evidenceProfileTable/IntegrationForm.js

```diff
@@ -72,42 +72,41 @@ const IntegrationForm = observer(props => {
 
                 <div className="col-md-6">
                     <QuillTextInput
```

Comment:
I think this was said elsewhere, but we still need the general description field. We can check w/ the scientists if we need to.